data/index.js

```diff
@@ -0,0 +1,61 @@
+const { Client } = require('pg');
+
+const client = new Client({
+  // user: '',
+  host: 'localhost',
```

Comment:
Remove credentials to another file

Reply:
I have no credentials, but if I did, I would do that :)